include/openssl/e_os2.h

```diff
@@ -276,6 +276,18 @@ typedef unsigned __int64 uint64_t;
 #  endif
 # endif
 
+/* Format specifier for printing size_t */
+# if (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L) \
+      && !defined(__VMS)
+#  define OSSLzu  "zu"
+# else
+#  ifdef SIXTY_FOUR_BIT_LONG
+#   define OSSLzu PRIu64
```

Comment:
I think SIXTY_FOUR_BIT_LONG does something other than you expect. Some 64 bit platforms define SIXTY_FOUR_BIT and not SIXTY_FOUR_BIT_LONG. I'm not sure defining either one of them is even required, but I guess most actually do.

But the PRIu64 thing is also a C99 thing, so this doesn't make any sense.